src/classes/AffiliationMappingsHealthCheckVMapper.cls

```diff
@@ -770,4 +865,42 @@ public virtual with sharing class AffiliationMappingsHealthCheckVMapper {
 
         return duplicateRecordTypeIds;
     }
+
+    /**********************************************************************************
+    * @description Retrieves a set of duplicate lookup field model keys in affiliation mappings
+    * @param  affiliationMappingsModelsList A list of models representing affiliation mappings in EDA Settings
+    * @param  accountLookupsOnContactFieldModelsByName A map of lookup field model by name
+    * @param  accountLookupsOnContactFieldModelsByLabel A map of lookup field model by label
+    * @return A set of duplicate lookup field model keys
+    **********************************************************************************/
+    @TestVisible
+    private Set<String> getDuplicateLookupFieldModelKeys(
```

Comment:
You can preserve this method and use the name of the lookup field model. When lacking an Id, a name suffices as our ultimate goal is to retrieve a unique identifier for a lookup field model for a given sObjectField, which is Relationship Type in this case.